invoicing_app/management/commands/update_incomplete_tax_receipts.py

```diff
@@ -0,0 +1,104 @@
+from django.core.management.base import BaseCommand, CommandError
+from django.db import connections
+from invoicing_app.models import PaymentOptions, Event, TaxReceipt
+import logging
+from optparse import make_option
+
+
+class Command(BaseCommand):
+
+    help = ('Change status of the tax receipts that have every requirement from INCOMPLETE to PENDING')
+
+    def __init__(self, *args, **kwargs):
+        self.tax_receipts = []
+
+        base_requirements = (
+            "recipient_name",
+            "recipient_address",
+            "recipient_tax_identifier_number",
+        )
+        self.tax_to_po_requirement_dict = {
+            "recipient_name": "epp_name_on_account",
+            "recipient_address": "epp_address1",
+            "recipient_tax_identifier_number": "epp_tax_identifier",
+            "recipient_postal_code": "epp_zip",
+            "recipient_city": "epp_city",
+            "tax_regime_type_id": "",
+        }
+        self.arg_requirements = base_requirements
+        self.br_requirements = base_requirements + ("recipient_postal_code",)
+        self.CPF_CHAR_COUNT_LIMIT = 11
+        super(Command, self).__init__(*args, **kwargs)
+
+    def handle(self, **options):
+        print("----Finding tax receipts that meet requiremente criteria----")
```

Comment:
We should be careful with the prints. Remember that in the old generation script, the logger didn't work in our local, but it did in stage. I think is better use a logger than a print.